holmes/plugins/prompts/investigation_procedure.jinja2

```diff
@@ -0,0 +1,210 @@
+{% if investigation_id %}
```

Comment:
I am curious, why investigation_id is so important. It's an uuid across one interactive or non-interactive mode and marking on session.